packages/core/redux/eos/thunks.js

```diff
@@ -0,0 +1,356 @@
+  if (!entry) {
+    return null
+  }
+
+  return dispatch(EosActions.eosTxUpdate(
+    key,
+    address,
+    EosUtils.createEosTxEntryModel({
+      ...entry,
+      ...props,
+    }),
+  ))
+}
+
+export const executeEosTransaction = (wallet, quantity, to, memo = '') => async (dispatch) => {
+  const tx = {
+    from: wallet.address,
+    quantity,
+    to,
+    memo,
+  }
+  const entry = EosUtils.createEosTxEntryModel({ tx })
+  await dispatch(EosActions.eosTxCreate(entry))
+  dispatch(submitTransaction(entry))
+}
+
+const processTransaction = ({ entry }) => async (dispatch, getState) => {
+  await dispatch(signTransaction({ entry }))
+  const signedEntry = eosPendingEntrySelector(entry.tx.from, entry.key)(getState())
+  if (!signedEntry) {
+    // eslint-disable-next-line no-console
+    console.error('signedEntry is null', entry)
+    return // stop execute
+  }
+  return dispatch(sendSignedTransaction({
+    entry: signedEntry,
+  }))
+}
+
+const signTransaction = ({ entry }) => async (dispatch, getState) => {
+  try {
+    const state = getState()
+    const signer = getEosSigner(state)
+    const network = getSelectedNetwork()(state)
+    const networkConfig = EosUtils.getEOSNetworkConfig(network[BLOCKCHAIN_EOS])
+    const preparedTx = await EosUtils.prepareTransactionToOfflineSign(entry.tx, networkConfig)
+
+    // sign
+    const signedTx = await signer.signTransaction(preparedTx)
+
+    dispatch(EosActions.eosTxUpdate(entry.key, entry.tx.from, EosUtils.createEosTxEntryModel({
+      ...entry,
+      tx: { ...entry.tx },
+      signedTx,
+    })))
+
+  } catch (error) {
+    dispatch(eosTxStatus(entry.key, entry.tx.from, { isErrored: true, error }))
+    throw error
+  }
+}
+
+const sendSignedTransaction = ({ entry }) => async (dispatch, getState) => {
+  const state = getState()
+  dispatch(eosTxStatus(entry.key, entry.tx.from, { isPending: true }))
+
+  // eslint-disable-next-line
+  entry = eosPendingEntrySelector(entry.tx.from, entry.key)(state)
+  if (!entry) {
+    // eslint-disable-next-line no-console
+    console.error('entry is null', entry)
+    return notifyEosError('entry is null') // stop execute
+  }
+
+  try {
+    const network = getSelectedNetwork()(state)
+    const { httpEndpoint, chainId } = EosUtils.getEOSNetworkConfig(network[BLOCKCHAIN_EOS])
+    const eos = Eos({ httpEndpoint, chainId }) // create eos for push new signed tx
+    eos.pushTransaction(entry.signedTx.transaction, (error, transaction) => {
+      if (error) {
+        dispatch(eosTxStatus(entry.key, entry.tx.from, { isErrored: true, error }))
+        return notifyEosError(error)
+      }
+      dispatch(notifyEosTransfer(entry))
+      const { processed, transaction_id } = transaction
+      dispatch(EosActions.eosTxUpdate(
+        entry.key,
+        entry.tx.from,
+        EosUtils.createEosTxEntryModel({
+          ...entry,
+          processed,
+          hash: transaction_id,
+          isSent: true,
+          isMined: true,
```

Comment:
Is it OK, that isMined equal true here?
The same for the method `setEos` below.

Reply:
Yep. It is OK, because blocks created every seconds.